firebase.go

```diff
@@ -49,6 +49,7 @@ type Firebase struct {
 	url    string
 	params _url.Values
 	client *http.Client
+	name   string
```

Comment:
Was the intent to have this as an exported field that can be accessed by someone?